pkg/hintrunner/zero/zerohint_others_test.go

```diff
@@ -57,5 +57,246 @@ func TestZeroHintMemcpy(t *testing.T) {
 				check: varValueInScopeEquals("n", *feltUint64(1)),
 			},
 		},
+		"SearchSortedLower": {
+			{
+				operanders: []*hintOperander{
+					{Name: "array_ptr", Kind: fpRelative, Value: feltUint64(7)},
+					{Name: "elm_size", Kind: fpRelative, Value: feltInt64(0)},
+					{Name: "n_elms", Kind: uninitialized},
+					{Name: "key", Kind: uninitialized},
+					{Name: "index", Kind: uninitialized},
+				},
+				makeHinter: func(ctx *hintTestContext) hinter.Hinter {
+					return newSearchSortedLowerHint(
+						ctx.operanders["array_ptr"],
+						ctx.operanders["elm_size"],
+						ctx.operanders["n_elms"],
+						ctx.operanders["key"],
+						ctx.operanders["index"],
+					)
+				},
+				errCheck: errorTextContains("invalid value for elm_size. Got: 0"),
+			},
+			{
+				operanders: []*hintOperander{
+					{Name: "array_ptr", Kind: fpRelative, Value: feltInt64(8)},
+					{Name: "elm_size", Kind: fpRelative, Value: feltInt64(1)},
+					{Name: "n_elms", Kind: fpRelative, Value: feltInt64(1)},
+					{Name: "key", Kind: fpRelative, Value: feltInt64(1)},
+					{Name: "firstElement", Kind: apRelative, Value: feltInt64(0)},
+					{Name: "index", Kind: uninitialized},
+				},
+				makeHinter: func(ctx *hintTestContext) hinter.Hinter {
+					return newSearchSortedLowerHint(
+						ctx.operanders["array_ptr"],
+						ctx.operanders["elm_size"],
+						ctx.operanders["n_elms"],
+						ctx.operanders["key"],
+						ctx.operanders["index"],
+					)
+				},
+				errCheck: errorIsNil,
```

Comment:
Shouldn't you be checking the value of `index`?

Reply:
yeah, but we cannot have a `check` and `errCheck` at the same time, thats why. I made a bunch of others tests testing the value of `index` , the hint works properly